controller.py

```diff
@@ -83,249 +83,254 @@ def get_docbins(
+            doc_ock.post_embedding_failed(user_id, model)
             message = f"The data type {attribute_data_type} is currently not supported for embeddings. Please contact the support."
             notification.create(
-                request.project_id,
-                request.user_id,
+                project_id,
+                user_id,
                 message,
                 enums.Notification.ERROR.value,
                 enums.NotificationType.EMBEDDING_CREATION_FAILED.value,
                 True,
             )
             send_project_update(
-                request.project_id, f"notification_created:{request.user_id}", True
+                project_id, f"notification_created:{user_id}", True
             )
             raise Exception(message)
     except HTTPError:
         print(traceback.format_exc(), flush=True)
         notification.create(
-            request.project_id,
-            request.user_id,
-            f"Could not load model {request.config_string}. Please contact the support.",
+            project_id,
+            user_id,
+            f"Could not load model {model}. Please contact the support.",
             enums.Notification.ERROR.value,
             enums.NotificationType.EMBEDDING_CREATION_FAILED.value,
             True,
         )
         send_project_update(
-            request.project_id, f"notification_created:{request.user_id}", True
+            project_id, f"notification_created:{user_id}", True
         )
         embedding.update_embedding_state_failed(
-            request.project_id,
+            project_id,
             embedding_id,
             with_commit=True,
         )
         send_project_update(
-            request.project_id,
+            project_id,
             f"embedding:{embedding_id}:state:{enums.EmbeddingState.FAILED.value}",
         )
-        doc_ock.post_embedding_failed(request.user_id, request.config_string)
+        doc_ock.post_embedding_failed(user_id, model)
         return status.HTTP_422_UNPROCESSABLE_ENTITY
 
     try:
         record_ids, attribute_values_raw = record.get_attribute_data(
-            request.project_id, attribute_name
+            project_id, attribute_name
         )
         embedding.update_embedding_state_encoding(
-            request.project_id,
+            project_id,
             embedding_id,
             with_commit=True,
         )
         send_progress_update_throttle(
-            request.project_id,
+            project_id,
             embedding_id,
             enums.EmbeddingState.ENCODING.value,
             initial_count,
         )
         send_project_update(
-            request.project_id,
+            project_id,
             f"embedding:{embedding_id}:state:{enums.EmbeddingState.ENCODING.value}",
         )
-        doc_ock.post_embedding_encoding(request.user_id, request.config_string)
+        doc_ock.post_embedding_encoding(user_id, model)
```

Comment:
we had some small issues with doc_ock in the barmenia project. Changes I mention aren't meant for here but the underlying _post_event method. (similar changes are in refinery-gateway)

To prevent issues if doc ock has some problems I'd recommend calling ock in a thread (so timeouts don't block calculation) and with a global try catch so the process isn't stopped if an error occurs